bootbox.js

```diff
@@ -216,6 +207,24 @@
    * merge a set of default dialog options with user supplied arguments
    */
   function mergeArguments(defaults, args, properties) {
+    var mapArgumentsResult = mapArguments(
+      args,
+      properties
+    );
+    if (defaults && defaults.buttons && mapArgumentsResult && mapArgumentsResult.buttons) {
```

Comment:
personally I like that code is readable, so I would add empty lines on these.

```
);
if (...)
```

to
```
);

if (...)
```